cli/src/commands/runTests.js

```diff
@@ -99,10 +99,6 @@ async function getOrderedFlowBinVersions(): Promise<Array<string>> {
       const QUERY_PAGE_SIZE = 100;
       const OS_ARCH_FILTER_RE = new RegExp(BIN_PLATFORM);
 
-      if (!await fs.exists(BIN_DIR)) {
```

Comment:
~What's the reason for removing this check?~ Nevermind, I see you moved it below.